runner/lib/tasks/testnet.js

```diff
@@ -35,6 +35,7 @@ const stateDir = '_agstate/agoric-servers';
 const profileName = 'testnet';
 const CLIENT_PORT = 8000;
 const clientStateDir = `${stateDir}/${profileName}-${CLIENT_PORT}`;
+const VerboseDebugEnv = 'SwingSet:vat,SwingSet:ls';
```

Comment:
```suggestion
const VerboseDebugEnv = 'agoric';
```

Reply:
I had copied the values from Agoric CLI, without knowing.